libs/pika/async_cuda/tests/unit/cublas_matmul.cpp

```diff
@@ -141,8 +139,8 @@ void matrixMultiply(pika::cuda::experimental::cuda_scheduler& cuda_sched,
 
     // Fill A and B with random numbers
     auto randfunc = [](T& x) { x = gen() / (T) RAND_MAX; };
-    pika::for_each(par, h_A.begin(), h_A.end(), randfunc);
-    pika::for_each(par, h_B.begin(), h_B.end(), randfunc);
+    std::for_each(h_A.begin(), h_A.end(), randfunc);
```

Comment:
Unfortunately no, at least not without ifdefs. libc++ doesn't support the execution policies yet, and I don't think it's worth adding an ifdef to be able to use `par` with libstdc++ (especially since it requires making sure that libstdc++ is actually configured with TBB to make use of the `par`).